lib/workers/repository/process/limits.ts

```diff
@@ -12,7 +12,7 @@ export async function getPrHourlyRemaining(
 ): Promise<number> {
   if (config.prHourlyLimit) {
     try {
-      logger.debug('Calculating hourly PRs remaining');
+      logger.debug('Calculating hourly PRs remaining (${config.prHourlyLimit})');
```

Comment:
```suggestion
      logger.debug(`Calculating hourly PRs remaining (limit: ${config.prHourlyLimit})`);
```

Reply:
I think the value should be logged as metadata

Reply:
Thanks @rarkins, disadvantage of just using GH UI to do PRs :D